commcare_connect/templates/tailwind/base.html

```diff
@@ -0,0 +1,34 @@
+{% load static i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CommCareConnect</title>
+    {% load static %}
+    {% block css %}
+       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/flatpickr/dist/flatpickr.min.css">
+       <link rel="stylesheet" href="{% static 'bundles/css/tailwind.css' %}">
+       <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/aquawolf04/font-awesome-pro@5cd1511/css/all.css" />
+       {% endblock %}
+       {% block javascript %}
+       <script src="{% static 'bundles/js/vendors-bundle.js' %}" defer></script>
+       <script src="{% static 'bundles/js/tailwind-bundle.js' %}" defer></script>
+       <script src="https://unpkg.com/htmx.org/dist/ext/loading-states.js"></script>
+       <script src="https://cdn.jsdelivr.net/npm/flatpickr"></script>
```

Comment:
These script imports can be remove as these are already present in the vendors bundle.